.github/ISSUE_TEMPLATE/BUG_REPORT.yml

```diff
@@ -0,0 +1,134 @@
+name: 🐞 Bug Report
+description: File a bug report if something is not working as expected.
+labels: []
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Before getting started, make sure you are in the right place by
+        ensuring that filing a bug report is the correct course of
+        action for your issue.
+
+        * For general technical questions, post a question on
+        [StackOverflow](http://stackoverflow.com/) with the `firebase` tag.
+
+        * For backend, console, and other non-SDK issues, reach out to
+        [Firebase Support](https://firebase.google.com/support/) for help.
+
+        If your issue does not fall under the above categories and instead
+        involves code in this repository not working as expected, please
+        complete and submit this bug report. Thank you!
+  - type: textarea
+    attributes:
+      label: Description
+      description: |
+        Describe the issue. Explain what you _expected_ to happen and what
+        _actually_ happened.
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: Reproducing the issue
+      description: |
+        Please provide either **steps to reproduce** or a [**minimal reproducible example**](https://stackoverflow.com/help/minimal-reproducible-example).
+        Providing a minimal reproducible example will help us triage your issue
+        faster.
+    validations:
+      required: false
+  - type: input
+    attributes:
+      label: Firebase SDK Version
+      description: What version of Firebase has this issue?
+      placeholder: "9.4.1"
+    validations:
+      required: true
+  - type: input
+    attributes:
+      label: Xcode Version
+      description: What version of Xcode is being used?
+      placeholder: "13.3.1"
+    validations:
+      required: true
+  - type: dropdown
+    attributes:
+      label: Installation Method
+      description: How did you add Firebase to your project?
+      options:
+        - Swift Package Manager
+        - CocoaPods
+        - Zip
+        - Carthage
+    validations:
+      required: true
+  - type: dropdown
+    attributes:
+      label: Firebase Product(s)
+      description: Which Firebase products are being used?
+      multiple: true
+      options:
+        - AB Testing
+        - Analytics
+        - App Check
+        - App Distribution
+        - Authentication
+        - Crashlytics
+        - Database
+        - DynamicLinks
+        - Firestore
+        - Functions
+        - In App Messaging
```

Comment:
nit: `In-App`

Reply:
Done in [7c34f5b](https://github.com/firebase/firebase-ios-sdk/pull/10119/commits/7c34f5b3fa726ec7bbcb6930b181a3ef04a22f00)